apis/v1alpha3/providerconfig_types.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
```

Comment:
Should we introduce `ProviderConfig` at `v1alpha1`?

Reply:
I think there might even be a case for introducing it at v1beta1, given that it's an evolution of the Provider type which has been stable for some time.

Reply:
That seems pretty reasonable to me

Reply:
Makes sense. The fields are pretty lean already (I don't think we'll ever get rid of cred secret ref), so, if we want to add new functionality we can just expand this without breaking anyone.